exchanges/gateio/gateio_websocket_delivery_futures.go

```diff
@@ -92,6 +90,9 @@ func (g *Gateio) GenerateDeliveryFuturesDefaultSubscriptions() (subscription.Lis
 				params["interval"] = "0"
```

Comment:
May I request that all the different asset types on `GenerateSubs` that don't support templating have a `// TODO`? Was the only thing that threw me

Reply:
Pushed sir 🫡 [dcea496](https://github.com/thrasher-corp/gocryptotrader/pull/1863/commits/dcea4961b411fe4157914dd8b8011e857f267d5a)

Reply:
Commentary [a2ed470](https://github.com/thrasher-corp/gocryptotrader/pull/1863/commits/a2ed4709b086c654bf02fe1755ede89b8541ce74)